__tests__/app.test.js

```diff
@@ -0,0 +1,24 @@
+const db = require('../db/connection');
+const request = require('supertest');
+const app = require('../app');
+const seed = require('../db/seeds/seed');
+const { articleData, commentData, topicData, userData } = require('../db/data/test-data/index');
+
+beforeEach(() => {
+    return seed({ topicData, userData, articleData, commentData });
+});
+
+afterAll(() => {
+    db.end();
+});
+
+describe("GET /api/topics", () => {
+    test("Should respond with an array of response objects, each object having properties of slug and description", () => {
+      return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(topicData);
```

Comment:
Although this current test passes, I wouldn't recommend testing anything that comes from a database, in this way - as we know that when info comes back from the database it may contain ids etc. You should instead test this in a way that checks if all of the objects in the received array does contain the properties you expect it to :)